tests/sweep_framework/sweep_utils/sharding_utils.py

```diff
@@ -9,10 +9,10 @@
 from tests.tt_eager.python_api_testing.sweep_tests.generation_funcs import _gen_reshape_args_from_volume
 
 
-def gen_sharded_spec_unary(num_shapes, max_tensor_size=4 * 1024 * 1024, layouts=["TILE_LAYOUT", "ROW_MAJOR_LAYOUT"]):
+def gen_sharded_spec_unary(num_shapes, max_tensor_size_per_core=62 * 1024, layouts=["TILE_LAYOUT", "ROW_MAJOR_LAYOUT"]):
```

Comment:
i think 62 is typo (4 * 1024 / 8 / 8 = 64)

Reply:
It's intentional. For 64 memory gets all used up and we start getting crashes.